src/WebUI/Components/PodStatusComponent.tsx

```diff
@@ -16,8 +18,10 @@ export class PodStatusComponent extends BaseComponent<IPodStatusProps, {}> {
         return (
             <div className="kube-status-container">
                 {
-                    this.props.statusProps &&
-                    <Status {...this.props.statusProps} size={this.props.statusSize || StatusSize.m} />
+                    this.props.toolTipText ?

```

Comment:
PR 54 has some changes on status, we should discuss, what would be the better modal, i was using Tooltip from tooltipex in the UI. we can close on this.

Reply:
@madhuv-msft Updated PR with changes as discussed.